src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/controller/UserController.java

```diff
@@ -83,8 +84,8 @@ public UserGetDto updateUser(@PathVariable UUID id, @RequestBody UserPutDto user
     // delete a specific user
     @DeleteMapping("/users/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void deleteUser(@PathVariable UUID id, @RequestBody UserPutDto userPutDto) {
-        User userInput = UserMapper.INSTANCE.userPutDtoToUser(userPutDto);
+    public void deleteUser(@PathVariable UUID id, @RequestBody UserPostPutDto userPostPutDto) {
+        User userInput = UserMapper.INSTANCE.userPostPutDtoToUser(userPostPutDto);
 
```

Comment:
User user = userService.findUserID(id)
            .orElseThrow(
                () -> new ResponseStatusException(HttpStatus.NOT_FOUND, "User couldn't be found with that user ID."));